Sources/StreamChat/Database/DTOs/ChannelDTO.swift

```diff
@@ -224,6 +226,13 @@ extension NSManagedObjectContext {
         }
 
         dto.isFrozen = payload.isFrozen
+        
+
+        // Backend only returns a boolean for blocked 1:1 channels
+        // on channel list query
+        if let isBlocked = payload.isBlocked {
+            dto.isBlocked = isBlocked
+        }
```

Comment:
Thinking if we should have 
```
} else {
  dto.isBlocked = false
}
```
for cases where one has 1:1 channel with a blocked user but then adds more members.

Reply:
Sounds reasonable. Updated 👍